DEVELOPERS.md

```diff
@@ -37,12 +39,54 @@ _Note:_ you will need the [Bitwarden CLI tool](https://bitwarden.com/help/articl
 - **Pip**
 - **Node.js v16.x** ([fnm](https://github.com/Schniz/fnm#installation) is recommended)
 - **npm v7.x**
+- **Postgres**
 
 #### `just` commands
 
 Each `just` command sets up a dev environment as part of running it.
 If you want to maintain your own virtualenv make sure you have activated it before running a `just` command and it will be used instead.
 
+
+#### Postgres
+
+##### Installing on macOS
+
+[Postgres.app](https://postgresapp.com/) is the easiest way to run Postgres on macOS, you can install it from homebrew (casks) with:
+
+  brew install --cask postgres-unofficial
+
+You will need to add its bin directory to your path for the CLI tools to work.
+
+Postico is a popular GUI for Postgres:
+
+  brew install --cask postico
```

Comment:
oooh, interesting, at the last count I was using pg_admin for a macOS GUI (& the new versions  **really** suck since they moved to the webserver arch)